test/snippet/range/view/take_line_adaptor_def.cpp

```diff
@@ -0,0 +1,6 @@
+#include <seqan3/std/ranges>
+
+int main()
+{
+    ranges::view::take_while([] (auto const & l) { return (l != '\r') && (l != '\n'); });
```

Comment:
```suggestion
    std::view::take_while([] (auto const & l) { return (l != '\r') && (l != '\n'); });
```

And what does this snippet demonstrate? Nothing happens...

Reply:
not resolved